proto/p4/v1/p4runtime.proto

```diff
@@ -447,6 +447,13 @@ message PacketReplicationEngineEntry {
   }
 }
 
+// A backup replica used as a fallback port for a replica.
+message BackupReplica {
+  // Added in v1.5.0.
```

Comment:
Please move the "Added in" comment to the right place, i.e. to the top of the message.